build/Dockerfile

```diff
@@ -1,18 +1,36 @@
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.11 AS builder
-ENV OPERATOR_PATH=/go/src/github.com/openshift/rbac-permissions-operator
-COPY . ${OPERATOR_PATH}
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
+
+# Allow specifying a GOPROXY cache during build to speed up dependency resolution
+ARG GOPROXY=https://proxy.golang.org
+
+ENV OPERATOR_PATH=/go/src/github.com/openshift/rbac-permissions-operator \
+    GO111MODULE=on \
+    GOPROXY=$GOPROXY
+
+RUN mkdir -p ${OPERATOR_PATH}
 WORKDIR ${OPERATOR_PATH}
-ENV GO111MODULE=on
-RUN \
-  cd ${OPERATOR_PATH} && \
-  go mod download && \
-  make gobuild
+COPY go.mod go.sum ./
+RUN go mod download
```

Comment:
Strictly speaking I don't think this is required since I think Go will attempt to download these modules as part of `make gobuild` on line 15?